src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow weak
```

Comment:
I have no idea what flow weak does, can you educate me? Also what isn't working with normal flow?